cluster-scope/base/operators.coreos.com/subscriptions/openshift-distributed-tracing-platform/subscription.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: jaeger-product
+  name: distributed-tracing-platform-product
   namespace: openshift-distributed-tracing
 spec:
   channel: stable
   installPlanApproval: Automatic
-  name: jaeger-product
+  name: distributed-tracing-platform-product
```

Comment:
seems like these should not be changed, the package names are required to be `jaeger-product` in the `redhat-operators` catalog.

Reply:
updated - all jaeger configuration remains the same